modules/cluster-logging-collector-tolerations.adoc

```diff
@@ -1,39 +1,43 @@
 // Module included in the following assemblies:
 //
-// * logging/cluster-logging-collector.adoc
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-collector-tolerations_{context}"]
-= Using tolerations to control the log collector pod placement
+= Using taints and tolerations to control log collector pod placement
 
-You can ensure which nodes the logging collector pods run on and prevent
-other workloads from using those nodes by using tolerations on the pods.
+You can specify which nodes the logging collector pods run on, and prevent other workloads from using those nodes, by configuring taints and tolerations.
```

Comment:
I realize the verbiage is unchanged but its worth considering a change.  I would defer to our general documentation here only because this IMO sort of reads like... you configure the collector and it keeps other services off a node when in fact I believe the principle is you ... taint a node and then configure your pod to tolerate the taint

Reply:
I'll update to be in line with the content at https://docs.openshift.com/container-platform/4.14/nodes/scheduling/nodes-scheduler-taints-tolerations.html

Reply:
I will create a follow up issue for this I think. The scope of the original issue was just to clean up the collector docs related to this, but it's clear that the conceptual information etc needs to be collated and improved in light of the link I posted above in the core docs. I don't want to block this PR on updating things that are outside the scope.

Reply:
Follow up issue to update conceptual docs and structure: https://issues.redhat.com/browse/OBSDOCS-760